inputfiles/removedTypes.jsonc

```diff
@@ -83,19 +83,6 @@
                     }
                 }
             },
-            "Response": {
-                "methods": {
-                    "method": {
-                        // BCD's api.Response.json is not about the static method but about Body#json.
```

Comment:
Only chrome/blink has this new API AFAIS. 
So it should not be part of typescript right now as we need two supporting engines.

Reply:
Hello!.
It's not just Chrome: Node.js, Cloudflare, Deno, Vercel's Edge runtime also support this static method.
These built-in types can not be extended (Response is a variable, it can not be extended), so it's a big blocker here.

Reply:
https://github.com/microsoft/TypeScript-DOM-lib-generator#why-is-my-fancy-api-still-not-available-here

Reply:
This is fair. But because the there is no "constructor" type to be extended, it is impossible to "augment" the constructor in user land (it's only possible to add instance methods, not static methods).

This is making it impossible for users the work around the limitation <picture data-single-emoji=":sad:" title=":sad:"><img class="emoji" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..Septfdz8f-2iBaXK.xuAIaPqJrxWBkxCOsEWxKXJWAIts9EfOAU71-ipfCj7DT2kGKm1G_Qm9zksTjWhaAjJp6AhaAY_gQAGOG9IMtM1naxXjPEItJQRwFOwgjsLgBguIbhKoPQ.X1U_h0yuzO0VxTfkoUGKgQ" alt=":sad:" width="20" height="auto" align="absmiddle"></picture> 
Do you have any suggestion I could implement?

Reply:
I suggested adding Node and Deno to the list of vendors: https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1476. If this proposal is accepted, the addition of the `static Response.json(data, init)` method will be allowed.

Reply:
Yeah, seems like since responses are more commonly generated on the backend, having multiple backend runtimes supporting this seems like it should be enough.

Reply:
@feugy in case this doesn't get accepted because of this (unusual) restriction about two **browser** engines implementing this, I also tagged you in the Mozilla / Gecko Bugzilla tracking issue over here (because you apparently have a local version of Firefox with this implemented):

https://bugzilla.mozilla.org/show_bug.cgi?id=1758943#c1